cdq-analysis/Dockerfile

```diff
@@ -6,5 +6,9 @@ COPY  . .
 
 RUN go mod download
 
-RUN go build -o ./main
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o cdq-analysis main.go
```

Comment:
It's probably better to use a separate builder stage and then use a smaller base image (like ubi minimal) for the runtime stage E.g.:

https://github.com/redhat-appstudio/application-service/blob/main/Dockerfile#L32

Reply:
updated